### PR TITLE
Styling

### PR DESCRIPTION
**Pull request by shannongamby**

### CODE REVIEW

File: app/controllers/posts_controller.rb [hunk 1]
```diff
@@ -3,12 +3,13 @@ class PostsController < ApplicationController
...
   def new
     @post = Post.new
+    @posts = Post.where({user: current_user}).order('created_at DESC')
   end
 
   def create
     @post = Post.create(post_params.merge({user_id: current_user.id}))
     flash[:danger] = "Post Added. Nobody cares.."
-    redirect_to posts_url
+    redirect_to new_post_path
```
Comment (runticle) [inline app/controllers/posts_controller.rb]: how come ?

File: app/views/posts/index.html.erb [hunk 2]
```diff
@@ -30,23 +30,23 @@
       <% end %>
       <%= link_to '💬', new_post_comment_path(post.id) %>
       <span class="pull-right">
-        <%= link_to like_post_path(post), method: 'post', remote: true do %> 👍
-          <span id="Post_<%= post.id %>_likes"><%= post.likes.count %>likes</span>
+        <%= link_to like_post_path(post), method: 'post', remote: true do %>
+          <span id="Post_<%= post.id %>_likes"><%= post.likes.count %> ⭐️</span>
```
Comment (runticle) [inline app/views/posts/index.html.erb]: ⭐️⭐️⭐️⭐️⭐️⭐️⭐️

File: app/views/posts/new.html.erb [hunk 3]
```diff
@@ -1,6 +1,56 @@
...
+          <%= link_to like_post_comment_path(post, comment), method: 'post', remote: true do %>
+            <span id="Comment_<%= comment.id %>_likes"><%= comment.likes.count %> ⭐️</span>
+          <% end %>
+        </span>
+        </div>
+      <% end %>
+    </div>
+  </div>
+<% end %>
+</div>
```
Comment (runticle) [inline app/views/posts/new.html.erb]: how come ?

**Comment by runticle** [comment]
great work team!

File: Gemfile.lock [hunk 4]
```diff
@@ -236,6 +236,8 @@ GEM
     thor (0.20.0)
     thread_safe (0.3.6)
     tilt (2.0.8)
+    time_difference (0.7.0)
```
Comment (imogenmisso) [inline Gemfile.lock]: We can delete this now, no longer being used I don't think?

File: Gemfile.lock [hunk 5]
```diff
@@ -285,6 +287,7 @@ DEPENDENCIES
   selenium-webdriver
   simplecov (~> 0.16.1)
   simplecov-console (~> 0.4.2)
+  time_difference (~> 0.7.0)
```
Comment (imogenmisso) [inline Gemfile.lock]: Can be removed here too

File: app/views/comments/like.js.erb [hunk 6]
```diff
@@ -1,4 +1,4 @@
 $(document).ready(function() {
-    $("#<%= @likeable.class %>_<%= @likeable.id %>_likes").html("<%= @likeable.likes.size %>likes");
+    $("#<%= @likeable.class %>_<%= @likeable.id %>_likes").html("<%= @likeable.likes.size %> ⭐️");
```
Comment (imogenmisso) [inline app/views/comments/like.js.erb]: Love this 🌟

File: app/views/posts/new.html.erb [hunk 7]
```diff
@@ -1,6 +1,56 @@
-<%= form_for @post do |form| %>
-  <%= form.label :message %>
-  <%= form.text_area :message %>
 
+<br>
+<div class="container" align="center">
+  <h2>
+    <%= current_user.name.capitalize %>'s profile!
```
Comment (imogenmisso) [inline app/views/posts/new.html.erb]: Nice addition of profile page

**Pull request merged**